airflow-core/src/airflow/settings.py

```diff
@@ -346,19 +361,24 @@ def configure_orm(disable_connection_pool=False, pool_class=None):
     log.debug("Setting up DB connection pool (PID %s)", os.getpid())
     engine_args = prepare_engine_args(disable_connection_pool, pool_class)
 
-    if conf.has_option("database", "sql_alchemy_connect_args"):
-        connect_args = conf.getimport("database", "sql_alchemy_connect_args")
-    else:
-        connect_args = {}
-
+    connect_args = _get_connect_args("sync")
```

Comment:
If we have hardcoded `sync` here, how do we get `async` args? Also the prepare_engine_args seems to be only for sync?

Reply:
Oh, seen